template/.eslintrc.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
     env: {
         browser: true
     },
-    plugins: ["@typescript-eslint", "react-hooks"],
```

Comment:
so this isn't compatible with preact?

Reply:
eslint-plugin-react-hooks is still enable.
Since the recommended settings were added in v3, I've changed it to use this.

eslint-plugin-react-hooks does not officially support preact.
However, eslint-plugin-react, which has been used since before this PR, also does not officially support preact.